FirstApp/FirstApp.tex

```diff
@@ -791,7 +791,7 @@ \subsection{Monticello packages}
 
 \subsection{Saving and loading code with Monticello.}
 On the left-hand side of the Monticello browser is a list of packages that have a version loaded into the image; packages that have been modified since they were loaded are marked with an asterisk.
-(These are sometimes referred to as \subind{package}{dirty} packages.)
+(These are sometimes referred to as \emph{\subind{package}{dirty} packages}.)

```

Comment:
Fair enough re the "dirty" vocabulary above. However, I would argue that the distance between this definition and the usage above is too large. Maybe repeat some form of explanation in the usage of the term "dirty" above?

Reply:
Good idea, thanks, added above (in Environment.tex)